src/main/io/osd.c

```diff
@@ -1346,6 +1350,19 @@ static void osdUpdateStats(void)
         }
     }
 #endif
+#ifdef USE_ESC_SENSOR
+    value = (escDataCombined->temperature * 10) / 10;
+    if (featureIsEnabled(FEATURE_ESC_SENSOR))
```

Comment:
This needs to go one on the outside of the value calculation. And the `{` needs to be on the same line.

Reply:
Not entirely sure what you meant. Hope what I did is correct. 

P.S. I'm still not back.